files/mcollective/pluginpackager/templates/aiomodule/README.md.erb

```diff
@@ -10,7 +10,8 @@
 
 <%= @plugin.metadata[:description] %>
 
-The <%= module_name %> module is based on the source from <%= @plugin.metadata[:url] %>.
+The <%= module_name %> module generated automatically, based on the source from <%= @plugin.metadata[:url] %>.
+You can find templates used for generation in https://github.com/choria-io/puppet-mcollective/tree/master/files/mcollective/pluginpackager/templates/aiomodule.
```

Comment:
And what do you think about moving it to the development section on the bottom of description? There we could add a link just to the repo, not to particular directory. something like:
```
Developement:

If you want to contribute to the module, you can find templates in https://github.com/choria-io/puppet-mcollective.
Mcollective plugin sources can be found in https://github.com/puppetlabs repos.
```